python/dependency_injection_container.py

```diff
@@ -0,0 +1,132 @@
+import json
+
+from dependency_injector import containers, providers
+from flask.globals import request
+
+from api.behaviors_handler import BehaviorsHandler
+from api.index_handler import IndexHandler
+from api.inventory_handler import InventoryHandler
+from api.message_handler import MessageHandler
+from lib.app_config import AppConfig
+from lib.behaviors.repository import Repository
+from lib.cli_parser import CliParser
+from lib.http_utils import HttpUtils
+from lib.validate.configuration import Configuration
+from repository.database_connection_info import DatabaseConnectionInfo
+from repository.database_inventory_repository import \
+    DatabaseInventoryRepository
+from repository.file_inventory_repository import FileInventoryRepository
+from repository.mysql_connector import MySqlConnector
+
+
+def inventory_repository_selector():
```

Comment:
Is it possible to pull this out of the main file and house it with other things related to the inventory?

Reply:
this method i feel iffy about since the selector is really only useful to the Container class below. it doesn't actually interact with or generate an inventory repository.

the get_inventory_data im on board with.

Reply:
ignore my comment above. ill move to inventory.

Reply:
Could we just move it to the bottom of the file, since it's just a supporting function to the Container itself?

Reply:
cant move it to the bottom since it has to be defined before its used (in container). but no worries, im moving it anyways.